finance/management/commands/xformnetfilerawdata.py

```diff
@@ -273,7 +273,7 @@ def load_f460A_row(row, agency, verbosity=1):  # noqa
     beneficiary.save()
 
     # Now we have all the parts, save!
-    money = models.IndependentMoney(  # or calculated_Amount
+    money, _ = models.IndependentMoney.objects.get_or_create(  # or calculated_Amount
```

Comment:
I understand. But why deal with "appears"? There are lots of ways Netfile (or ourselves) can have an error. In addition, the transaction is explicitly declared as unique on source & transaction ID. So, it just makes logical sense to select on the fields that make it unique.

I don't see any upside to adding extra fields to the query. It's more brittle and doesn't follow the explicit schema. Do you?